spikeextractors/extractors/klustaextractors/klustaextractors.py

```diff
@@ -37,13 +49,16 @@ def __init__(self, folder_path):
         self._kwargs = {'folder_path': str(Path(folder_path).absolute())}
 
 
+# noinspection SpellCheckingInspection
 class KlustaSortingExtractor(SortingExtractor):
     extractor_name = 'KlustaSortingExtractor'
     installed = HAVE_KLSX  # check at class level if installed or not
     installation_mesg = "To use the KlustaSortingExtractor install h5py: \n\n pip install h5py\n\n"  # error message when not installed
     is_writable = True
     mode = 'file_or_folder'
 
+    default_cluster_groups = {0: 'Noise', 1: 'MUA', 2: 'Good', 3: 'Unsorted'}
+
```

Comment:
It could be an option to have an `exclude_cluster_groups` argument similar to the [PhySortingExtractor](https://github.com/SpikeInterface/spikeextractors/blob/master/spikeextractors/extractors/phyextractors/phyextractors.py#L59). This can be used to exclude noise or unsorted units automatically (see [here](https://github.com/SpikeInterface/spikeextractors/blob/master/spikeextractors/extractors/phyextractors/phyextractors.py#L131-L140)). What do you think?

Reply:
yes that's useful, added it